.env.prod

```diff
@@ -1 +1 @@
-CONSUL_HOST="internal-alb-hfa-us-e-1-p-int-vpc-gen-834019598.us-east-1.elb.amazonaws.com"
+CONSUL_HOST="consul.sesac.com"
```

Comment:
Did Eric make these changes in Prod? As of yesterday Ian told me he had not. Also, do you need `https://consul...` here?

Reply:
hmm good question. @matchbookmac should I hold off on adjusting this in PROD for now?

Reply:
@MattRooney This get interpolated [here](https://github.com/sesac/shipshape/blob/master/bin/provision_by_env#L125). http should re-direct to https.

Reply:
Looks like I was wrong about that. Seems like he's got it going now
https://consul.sesac.com/v1/catalog/service/consul